package.json

```
@@ -32,7 +32,7 @@
     "eth-ens-namehash": "^2.0.8",
     "eth-json-rpc-errors": "^2.0.2",
     "eth-json-rpc-infura": "^4.0.1",
-    "eth-keyring-controller": "^5.3.0",
+    "eth-keyring-controller": "5.6.0",
```

It would be better to use a range here, so we can update it in the extension and mobile without having to update this as well
```suggestion
    "eth-keyring-controller": "^5.6.0",
```